### PR TITLE
Added Links & Fixed Samples files to cart function in Custodian tables in CaseDetail Page & ArmsDetail Page.

### PR DESCRIPTION
**Pull request by iksheth**
## Description
Added Links & Fixed Samples files to cart function in Custodian tables in CaseDetail Page & ArmsDetail Page.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
Yes on local

### CODE REVIEW

**Comment by skiran86** [comment]
@iksheth Just like you implemented hyperlink ability on the case detail page, can you do the same thing on the arm detail page

**Comment by iksheth** [comment]
@skiran86 implemented.

**Review by skiran86 — APPROVED** [review]

File: src/bento/caseDetailData.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
+/* eslint-disable no-unused-vars */
```
Comment (AjayDoddapaneni) [inline src/bento/caseDetailData.js:1]: Are you seeing any issues due to this?

**Comment by AjayDoddapaneni** [comment]
Small comment otherwise good to go!!!

**Pull request merged**